aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -11,7 +23,7 @@ Parameters:
 <% if database -%>
   DatabaseUsername:
     Type: String
-    Default: master
+    Default: masterzoom
```

Comment:
typo

Reply:
Fixed. Still getting used to having multiple monitors. Was searching for the Zoom app on a different monitor but the active/foreground window was my IDE 😳